src/epub/text/freedom.xhtml

```diff
@@ -180,7 +180,7 @@
 			<p>She was impatient with him. “You don’t understand at all. See here, why don’t you come along tonight? I’m sure I can get you in.”</p>
 			<p>“It sounds like a good idea,” Ilya Simonov said. He was completely sober now.</p>
 			<hr/>
-			<p>He made a written report to Kliment Blagonravov before turning in. He mentioned the rather free discussion of matters political in the Czech capital, using the man he’d met in the beer hall as an example. He reported⁠—although, undoubtedly, Blagonravov would already have the information⁠—hearing of a Polish Tri-D film which had defended the Old Bolsheviks purged in the 1930s. He mentioned the literary magazine, with its caricature of Frol Zverev, and, last of all, and then after hesitation, he reported party member Catherina Panova, who evidently belonged to a group of intellectuals who were not above listening to a talk given by a foreign journalist who was not speaking under the auspices of the Czech Party nor the government.</p>
+			<p>He made a written report to Kliment Blagonravov before turning in. He mentioned the rather free discussion of matters political in the Czech capital, using the man he’d met in the beer hall as an example. He reported⁠—although, undoubtedly, Blagonravov would already have the information⁠—hearing of a Polish <abbr epub:type="z3998:acronym">TriD</abbr> film which had defended the Old Bolsheviks purged in the 1930s. He mentioned the literary magazine, with its caricature of Frol Zverev, and, last of all, and then after hesitation, he reported party member Catherina Panova, who evidently belonged to a group of intellectuals who were not above listening to a talk given by a foreign journalist who was not speaking under the auspices of the Czech Party nor the government.</p>
```

Comment:
Keep the dash in these cases as well, and I don't think Tri-D is an acronym. It does not require `<abbr>`.

Reply:
fixed